editor/editor_node.cpp

```diff
@@ -901,6 +895,20 @@ void EditorNode::_update_update_spinner() {
 	OS::get_singleton()->set_low_processor_usage_mode(!update_continuously);
 }
 
+void EditorNode::_execute_upgrades() {
+	if (run_surface_upgrade_tool) {
```

Comment:
I know this is unrelated, but I was curious about why these checks are excluding. I've found it's only possible that one of them is instructed to run. If only this has been modeled as an enum... But again, not in this PR's scope.

Reply:
I plan to eventually replace both tools with a universal one. They do pretty much the same thing.